app/controllers/partners/requests_controller.rb

```diff
@@ -12,6 +12,12 @@ def new
       @partner_request.item_requests.build
 
       @requestable_items = PartnerFetchRequestableItemsService.new(partner_id: current_partner.id).call
+      if Flipper.enabled?(:enable_packs)
+        # hash of (item ID => hash of (request unit name => request unit plural name))
+        @item_units = current_partner.organization.items.to_h do |i|
+          [i.id, i.request_units.to_h { |u| [u.name, u.name.pluralize] }]
+        end
```

Comment:
Since this is identical to the one in `create` please extract it into a private method.